File: leet_2194_cells_in_range_on_excel.py
class Solution:
    def cellsInRange(self, s: str) -> List[str]:
        a = list(s)
        c1 = a[0]
        c2 = a[3]
        n1 = int(a[1])
        n2 = int(a[-1])
        rest = []
        while ord(c1) < ord(c2) + 1:
            i = n1
            while i < n2 + 1:
                temp = c1 + str(i)
                rest.append(temp)
                i += 1
            c1 = chr(ord(c1) + 1)
        return rest

File: leet_2194_cells_in_range_on_excel.cpp
class Solution {
public:
    vector<string> cellsInRange(string s)
    {
        vector<string>  res;
        stringstream    ss(s);
        char            c1, c2;
        int             n1, n2;

        ss >> c1 >> n1 >> c2 >> c2 >> n2;
        while (c1 < c2 + 1)
        {
            int i = n1;
            while (i < n2 + 1)
            {
                string  temp;
                temp += c1;
                temp += i + 48;
                res.push_back(temp);
                ++i;
            }
            ++c1;
        }
        return res ;
    }
};
